html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label>
+    <div>
+      <input id="email" type="text" name="email">
+    </div>
+    Why is coding cool?
+    <div>
+      <select name="codingreasons">
```

Comment:
`select` can also be tied with `label` so when user clicks on `Why is coding cool?` then it focus on the `select` element